python/cuml/cuml/dask/manifold/umap.py

```diff
@@ -44,7 +44,7 @@ class UMAP(BaseEstimator, DelayedTransformMixin):
         >>> X, y = make_blobs(1000, 10, centers=42, cluster_std=0.1,
         ...                   dtype=np.float32, random_state=10)
 
-        >>> local_model = UMAP(random_state=10)
+        >>> local_model = UMAP(random_state=10, verbose=0)
```

Comment:
We now must set this because the default logging level will result in an info log that was previously being incorrectly suppressed.